janus_core/helpers/utils.py

```diff
@@ -3,6 +3,42 @@
 from pathlib import Path
 from typing import Optional
 
+from ase import Atoms
+from spglib import get_spacegroup
+
+
+def spacegroup(
+    struct: Atoms, sym_tolerance: float = 0.001, angle_tolerance: float = -1.0
+) -> str:
+    """
+    Determine the spacegroup for a structure.
+
+    Parameters
+    ----------
+    struct : Atoms
+        Structure as an ase Atoms object.
+    sym_tolerance : float
+        Atom displacement tolerance for spglib symmetry determination.
+        Default 0.001 Å.
+    angle_tolerance : float
+        Angle precision for spglib symmetry determination.
+        Default: -1.0 in degrees.
```

Comment:
```suggestion
        Atom displacement tolerance for spglib symmetry determination, in Å.
        Default is 0.001.
    angle_tolerance : float
        Angle precision for spglib symmetry determination, in degrees. Default is -1.0,
        which means an internally optimized routine is used to judge symmetry.
```

Reply:
Should note if this change is made should also be done in `geom_opt.py`